app/controllers/address.js

```diff
@@ -0,0 +1,3 @@
+const crudController = require('./crudController');
+
+module.exports = crudController('address');
```

Comment:
I would create a class `AddressController` which extends the base controller, then `module.exports = new AddressController('address');`.